packages/components/src/internal/util/utils.ts

```diff
@@ -673,7 +673,7 @@ export function isQuotedWithDelimiters(value: any, delimiter: string): boolean {
     }
 
     const strVal = value + '';
-    if (strVal.indexOf(delimiter) === -1) return false;
+    if (strVal.indexOf(delimiter) === -1 && value.indexOf('\r') === -1 && value.indexOf('\n') === -1) return false;
```

Comment:
since this "value has newline char" check is added in a couple of places, maybe it could be factored out into a function whose name can work as partial documentation (i.e. `hasNewlineChar(value)`)

Reply:
Added comment that explains the logic and rewrote the code to match server side better.